README.md

```diff
@@ -150,6 +151,41 @@ Here's how the previous example would be written out using `ix-path` and `ix-par
 
 **Please note**: `ix-params` must be a valid JSON string. This means that keys and string values must be surrounded by double quotes, e.g., `"fit": "crop"`.
 
+### `ix-sizes` attribute
+
+Creates an `img` tag with the `sizes` attribute automatically updating to match the image's display size.
```

Comment:
This is really nitpicky but the "Creates an `img` tag" might read a little confusing because it's an attribute on an existing `img` tag? Although the same issue exits under the `ix-src` [section](https://github.com/imgix/imgix.js/pull/300/files#diff-b335630551682c19a781afebcf4d07bf978fb1f8ac04c6bf87428ed5106870f5R97)."

How about:
```suggestion
When set to `auto`, automatically updates an `img` tag's `sizes` attribute to match the image's display size.
```

Reply:
On a separate note, does "`ix-sizes` only work with `ix-src`/`ix-path`? If so, we should call that out.